### PR TITLE
Add support for on premise DevOps

### PR DESCRIPTION
**Pull request by bmdixon**
Adds a new `BaseUrl` setting value and uses this in place of the previously hard coded 'dev.azure.com'. This allows the configuration of an on-premise instance (i.e. Azure DevOps Server).

### CODE REVIEW

**Comment by panuoksala** [comment]
Yes this looks very good. Maybe we should change the Organization name into Organisation URL?

File: property_inspector/property_inspector.html [hunk 1]
```diff
@@ -13,9 +13,13 @@
             and
             Elgato SDK Documentation -> https://developer.elgato.com/documentation/stream-deck/sdk/property-inspector/
     -->
+        <div class="sdpi-item" id="base_url" type="field">
+            <div class="sdpi-item-label">DevOps base URL</div>
+            <input id="txtBaseUrl" class="sdpi-item-value" type="text" placeholder="Base URL (e.g. dev.azure.com)" onKeyUp="setSettings(event.target.value, 'BaseUrl')" required />
+        </div>
         <div class="sdpi-item" id="organization_name" type="field">
             <div class="sdpi-item-label">Organization name</div>
```
Comment (panuoksala) [inline property_inspector/property_inspector.html:21]: Organization URL

**Comment by bmdixon** [comment]
> Yes this looks very good. Maybe we should change the Organization name into Organisation URL?

Thanks, happy to make that change. Do you mean we should keep just one setting for the whole thing or just change the name of that field?

**Comment by panuoksala** [comment]
> > Yes this looks very good. Maybe we should change the Organization name into Organisation URL?
> 
> Thanks, happy to make that change. Do you mean we should keep just one setting for the whole thing or just change the name of that field?

Now we have the organization name and base url. You need to set both into something like "Testcompany" and "https://dev.azure.com/"

I think we should combine these into just URL and then the user has to fill only one field. I checked that plugin is currently using the organization name only for the URL, so it can be removed.

**Comment by panuoksala** [comment]
I will try to test this today and merge in if everything works as expected.

This is great improvement, thanks!

**Pull request merged**